code/src/components/AddTask.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import styled from "styled-components";
+import { tasks } from "../reducers/tasks";
+
+export const AddTask = () => {
+  const [item, setItem] = useState("");
+  const dispatch = useDispatch();
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    if (item !== "") dispatch(tasks.actions.addItem(item));
+    setItem("");
+  };
+
+  return (
+    <TaskForm onSubmit={handleSubmit}>
+      <NewTask>
+        <AddNewTask
+          type="submit"
+          value="+"
+          active={item.length > 0 ? true : false}
```

Comment:
Does this really need the ternary? 

This is like saying "If true return true else false". You could just use `active={item.length > 0}`